README.md

```diff
@@ -1,10 +1,23 @@
 # Design of integrated circuits - MAC Unit
-This MAC-unit was designed as part of a student project. Here you can view it's design and implemetation, and simulation results. Full project report can be read at: [PLACEHOLDER]. 
+This MAC-unit was designed as part of a student project. Here you can view its design, implementation, and simulation. The full project report can be read at: [PLACEHOLDER]. 
 
 ## Repository structure
-Primary directories are *spice* and *verilog*. Under spice you will find all circuit and model files used for transistor simulations. Verilog contains the design and verification of the hardware at gate and system level.
+The primary directories are *spice* and *verilog*. Under Spice, you will find all circuit and model files used for transistor simulations. Verilog contains the design and verification of the hardware at the gate and system level.
 
 - **verilog**
     - **simulations** - simulation results.
     - **src** - circuits implemented at gate level.
-    - **testbench** - testbenches for circuits.
\ No newline at end of file
+    - **testbench** - testbenches for all circuits.
+- **spice**
+    - **plots** - transistor simulation results at different process corners and temperatures.
+    - **simulations** - raw data generated from the simulations.
+    - **src** - Simulation code, transistor design of circuits, and model parameters used.
+
+ ## Running Verilog Code
+ The Verilog code can be run in Active-HDL or VS Code. Begin with cloning the repository do your desired directory:
+ ```
+git clone https://github.com/AasmundN/mac-unit.git
```

Comment:
Isn't it obvious that you have to clone the repository? think this can be removed.

Reply:
> Isn't it obvious that you have to clone the repository? think this can be removed.

Better to be too specific than too vague. Assume that the person reading knows to little.